demos/apps/bookinfo/bookinfo.yaml

```diff
@@ -63,6 +69,11 @@ spec:
   selector:
     app: ratings
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ratings-v1-service-account
```

Comment:
I mean: typically different versions of an app would have the same permissions / roles, and I would expect
ACLs/permissions to be set to the role independent of the release.